lib/components/embed.js

```diff
@@ -1,20 +1,26 @@
 import element from 'magic-virtual-element';
 import renderText from '../text';
 
-const defaultRenderCaption = text => <figcaption>{text}</figcaption>;
+const defaultRenderCaption = (text, attribution) => {
+  const attributionEl = attribution.length > 0 ? <cite>{attribution}</cite> : '';
+  return (<figcaption>
+    {text.concat([attributionEl])}
```

Comment:
I'm guessing it's because `children` of a component can be an array of elements or an element, but not a combination of both.